src/section/section.tsx

```diff
@@ -8,13 +8,54 @@ import { SnakeTail } from "../illustrations/snake-tail";
 import { Grid, GridColumn } from "../grid";
 import { Spacer } from "../spacer";
 import { Color } from "../types";
-import { SnakeLongNeck } from "../illustrations/snake-long-neck";
+import { Illustration } from "../illustrations/types";
+import { getIllustration } from "../illustrations/illustrations";
+
+const DX_ILLUSTRATIONS = [
```

Comment:
two things in my mind:

1. We would need to remember to update this list to use any new illustration we add
2. does `dx` mean anything in English?

point (1) made me think: why is the special case the fact that the illustration is on the right (which is the default). It makes more sense that the special case is when the illustration is on the left?

Reply:
but some other illustration like snakePencil, snakeTail, snakeLetter would be on left, so we don't want all the illustrations to go always on the right... 

DX maybe it's only for Italian, I thought was Latin or international but apparently is not.

Reply:
maybe we can put a property? 
rightIllustration / leftIllustration

Reply:
are you sure snakePencil, snakeTail, snakeLetter go on the left?

Reply:
no they are on the right, you are right... but they will need some customization anyway because they are not on 'the bottom of the component', they need to be on the top... so the snake seems like is coming from above...